Development

keshav-c opened the pull request:
Awaiting your comments. We haven't done much for the header and footer sections as per the project requirements.

We have created 2 pages, one representing the original and the other it's grey-scale heatmap as per our perception.

Both are linked together via hyper links in the header.

keshav-c committed:
Add project directory structure

keshav-c committed:
Add HTML structure

keshav-c committed:
Add featured articles section

mosesogwo committed:
latest-articles container in progress

mosesogwo committed:
curated articles container in progress

keshav-c committed:
Finish page structure

keshav-c committed:
Font selection

keshav-c committed:
Adjust fonts

Add typeface to body, adjust font sizes and positions of text
content. Add images to banners and boxes where it is helpful in
order to make a visual comparison

keshav-c committed:
Add heatmap, finish adjusting fonts

Finish adjusting fonts for the remaining sections of the page.
Use the header to link the original representation and it's
heatmap together. Add backgrounds to different page sections
giving weight to more prominent sections in term of visual
hierarchy.

keshav-c committed:
Minor changes before consolidating classes

keshav-c committed:
Replace universal selector in CSS

keshav-c committed:
Stickler corrections 1

keshav-c committed:
Stickler corrections 2

keshav-c committed:
Remove fixed height from conference panel

efrapp commented on styles/main.css:
```diff
@@ -0,0 +1,389 @@
+.featured-articles {
+    display: grid;
+    grid-auto-rows: 45em;
+    grid-template-columns: repeat(2, 1fr);
```
Don't worry about this error, it is something related to the [css lint](http://csslint.net/) Stickler uses. You can check a discussion about that here: https://github.com/CSSLint/parser-lib/issues/228

keshav-c replied:
Thanks. I guess w3 validators are going to be more useful from now on, if stickler can't even validate grid layouts.

efrapp commented on styles/main.css:
```diff
@@ -0,0 +1,389 @@
+    display: grid;
+    grid-auto-rows: 45em;
+    grid-template-columns: repeat(2, 1fr);
+    grid-template-rows: 5fr 4fr;
```
Same thing here and in all parts where this error appears.

efrapp reviewed (changes requested):
# Good Job! but I want you to address some things

The purpose of this project is to enforce design principles so you made a great job creating the grayscale heat map page and also the other page was great but is not needed for this project, you don't need to delete it it is just to put the focus in the grayscale page because there you are applying the key concepts for this project.

The things I want you to address are the followings:

1. Apply the correction shown in the HTML validator [here](https://validator.w3.org/nu/?doc=https%3A%2F%2Fkeshav-c.github.io%2Fsmashing-mag%2Findex.html). This is for the `index.html` file but it is important that you have those corrections in mind.
2. Apply the corrections shown in the HTML validator for the [`heatmap.html`](https://validator.w3.org/nu/?doc=https%3A%2F%2Fkeshav-c.github.io%2Fsmashing-mag%2Fheatmap.html) file.
3. Please update your GitHub Page link because in the CSS validator it shows errors that you already solved. See [here](http://jigsaw.w3.org/css-validator/validator?uri=https%3A%2F%2Fkeshav-c.github.io%2Fsmashing-mag%2Fstyles%2Fmain.css).

For point 1 and 2 pay more attention to the errors than the warnings.

When you are done with this changes, please, submit a new code review to check the changes.

keshav-c committed:
Heatmap focus, split font links

Linter doesn't recognise "|" used to include multiple fonts in the
same link tag. Also, make the heatmp the focus of the project by
making it the index, rather than the original clone. Add alts to
images where missing.

keshav-c committed:
main.css error correction

Replace min-max with legal function minmax

keshav-c commented:
I have made all corrections as suggested. Awaiting your re-review.

I have made the heatmap show up as index, so that is the focus and the original links from the heatmap page.

Also the warnings ask me to include section headings that are not a part of the original markup so I ended up ignoring those.

efrapp reviewed (approved):
# Ok you made a great job! 👍 

You can continue in your path to achieving the mastery!

You can submit now your completion form for this project.

I whish you the best.
